docs/core/compatibility/unsupported-apis.md

```diff
@@ -0,0 +1,319 @@
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.Protected?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.ProviderName?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.ProviderType?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.RandomlyGenerated?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.Removable?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.CspKeyContainerInfo.UniqueKeyContainerName?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.HashAlgorithm.Create?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HashAlgorithm.Create(System.String)?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HMAC.Create?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HMAC.Create(System.String)?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HMAC.HashCore%2A?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HMAC.HashFinal%2A?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.HMAC.Initialize%2A?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.KeyedHashAlgorithm.Create?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.KeyedHashAlgorithm.Create(System.String)?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.ProtectedData.Protect%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.ProtectedData.Unprotect%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Security.Cryptography.RSA.FromXmlString%2A?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.RSA.ToXmlString%2A?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.SymmetricAlgorithm.Create?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.SymmetricAlgorithm.Create(System.String)?displayProperty=nameWithType> | All |
+
+## System.Security.Cryptography.Pkcs
+
+| Member | Platform |
+| - | - |
+| <xref:System.Security.Cryptography.Pkcs.CmsSigner.%23ctor(System.Security.Cryptography.CspParameters)?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.Pkcs.SignedCms.ComputeSignature(System.Security.Cryptography.Pkcs.CmsSigner,System.Boolean)?displayProperty=nameWithType> | All |
+| <xref:System.Security.Cryptography.Pkcs.SignerInfo.ComputeCounterSignature?displayProperty=nameWithType> | All |
+
+## System.Security.Cryptography.X509Certificates
+
+| Member | Platform |
+| - | - |
+| <xref:System.Security.Cryptography.X509Certificates.X509Certificate.Import%2A?displayProperty=nameWithType> | All |
```

Comment:
we also have this exception documented for https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate.system-runtime-serialization-iserializable-getobjectdata?view=netframework-4.8&branch=master and
https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate.system-runtime-serialization-ideserializationcallback-ondeserialization?view=netcore-3.1&branch=master

https://source.dot.net/#System.Security.Cryptography.X509Certificates/System/Security/Cryptography/X509Certificates/X509Certificate.cs,184
https://source.dot.net/#System.Security.Cryptography.X509Certificates/System/Security/Cryptography/X509Certificates/X509Certificate.cs,189

It seems the constructor also throws
https://source.dot.net/#System.Security.Cryptography.X509Certificates/System/Security/Cryptography/X509Certificates/X509Certificate.cs,169

Reply:
I didn't document GetObjectData and OnDeserialization because they have the default accessibility level, which I can't figure out exactly [based on this doc](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/accessibility-levels), but seems like that might be private?